pages/docs/installation/installation-distribution.md

```diff
@@ -28,7 +28,7 @@ It comprises of the following components:
   - Matlab bindings: [v2.3.0.1](https://github.com/precice/matlab-bindings/releases/tag/v2.3.0.1)
   - Python bindings: [v2.3.0.1](https://github.com/precice/python-bindings/releases/tag/v2.3.0.1)
 - Adapters:
-  - CalculiX adapter: [v2.19.0](https://github.com/precice/calculix-adapter/releases/tag/v2.19.0) (first tagged release)
+  - CalculiX adapter: [v2.20.0](https://github.com/precice/calculix-adapter/releases/tag/v2.20.0) (second tagged release)
```

Comment:
This looks wrong, we should not update the versions we used in previous distributions.

Reply:
Yes, sorry. Fixed.